bigtable/client/internal/table_admin.h

```diff
@@ -114,6 +114,10 @@ class TableAdmin {
                         bigtable::ConsistencyToken const& consistency_token,
                         grpc::Status& status);
 
+  std::vector<::google::bigtable::admin::v2::Snapshot> ListSnapshots(
```

Comment:
Consider making the returned collection optional.  I know we hardcoded vector elsewhere, we should fix those too.

Reply:
As far as I know, std::optional was introduced in c++17. Do you still want me to add this?

Reply:
I meant optional as we did for SampleRowKeys.